tests/pipelines/kolors/test_kolors.py

```diff
@@ -20,12 +20,11 @@
 
 from diffusers import (
     AutoencoderKL,
-    ChatGLMModel,
-    ChatGLMTokenizer,
     EulerDiscreteScheduler,
     KolorsPipeline,
     UNet2DConditionModel,
 )
+from diffusers.pipelines.kolors import ChatGLMModel, ChatGLMTokenizer
```

Comment:
I think we already install `sentencepiece` for tests. So should be good:
https://github.com/huggingface/diffusers/blob/b1f43d71897ad2c73cb891d2e92d23bc7d46a4be/setup.py#L126

Reply:
yeah, that was the main reason I didn't catch this